pkg/reconciler/v1alpha1/service/resources/route.go

```diff
@@ -35,7 +37,9 @@ func MakeRoute(service *v1alpha1.Service) (*v1alpha1.Route, error) {
 			OwnerReferences: []metav1.OwnerReference{
 				*kmeta.NewControllerRef(service),
 			},
-			Labels: makeLabels(service),
+			Labels: resources.MakeLabels(service, map[string]string{
+				serving.ServiceLabelKey: service.Name,
+			}),
```

Comment:
+1 to @markusthoemmes comments, but I really like this cleanup from the perspective of the callsites, very clear.